### PR TITLE
Azure Stack Storage Administrator SDK

### PR DESCRIPTION
**Pull request by eridai**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@eridai__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (eridai). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

File: src/ResourceManagement/AzureStackStorageAdmin/NuGet.Config [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<settings>
+  <repositoryPath>..\..\packages</repositoryPath>
```
Comment (hovsepm) [inline src/ResourceManagement/AzureStackStorageAdmin/NuGet.Config]: Align this folder path with the rest of projects in the ResourceManagement folder. I think you are missing one more ..\

File: src/ResourceManagement/AzureStackStorageAdmin/StorageManagement/StorageManagement.csproj [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <Compile Include="Customization\StorageAdminManagementClient.Customization.cs" />
+    <Compile Include="Generated\StorageAdminManagementClient.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Hyak.Common">
+      <HintPath>$(LibraryNugetPackageFolder)\Hyak.Common.1.0.3\lib\portable-net403+win+wpa81\Hyak.Common.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Azure.Common">
+      <HintPath>$(LibraryNugetPackageFolder)\Microsoft.Azure.Common.2.0.3\lib\portable-net45+wp8+wpa81+win\Microsoft.Azure.Common.dll</HintPath>
```
Comment (hovsepm) [inline src/ResourceManagement/AzureStackStorageAdmin/StorageManagement/StorageManagement.csproj]: this is incorrect version should be 2.1.0

File: src/ResourceManagement/AzureStackStorageAdmin/StorageManagement/StorageManagement.csproj [hunk 3]
```diff
@@ -0,0 +1,45 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
```
Comment (hovsepm) [inline src/ResourceManagement/AzureStackStorageAdmin/StorageManagement/StorageManagement.csproj]: align entire csproj with the other projects - take a look to \src\ResourceManagement\Resource\ResourceManagement\ResourceManagement.csproj"

File: src/ResourceManagement/AzureStackStorageAdmin/StorageAdmin.Tests/StorageAdmin.Tests.csproj [hunk 4]
```diff
@@ -0,0 +1,58 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (hovsepm) [inline src/ResourceManagement/AzureStackStorageAdmin/StorageAdmin.Tests/StorageAdmin.Tests.csproj]: align with \src\ResourceManagement\Resource\Resources.Tests\Resources.Tests.csproj project

**Pull request merged**